publish.gradle

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2019 Cyface GmbH
+ *
+ * This file is part of the Cyface SDK for Android.
+ *
+ * The Cyface SDK for Android is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Cyface SDK for Android is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the Cyface SDK for Android. If not, see <http://www.gnu.org/licenses/>.
+ */
+/**
+ * Gradle's publish file for this module.
+ *
+ * @author Armin Schnabel
+ * @version 1.0.0
+ * @since 5.0.0-beta2
+ */
+
+configure(subprojects.findAll({ it.name in ["persistence", "synchronization", "datacapturing"] })) {
+    pr ->
+        apply plugin: 'com.android.library'
+        apply plugin: 'maven-publish'
+
+        publish.dependsOn 'assemble'
+
+        // Load credentials
+        def properties = new Properties()
+        properties.load(new FileInputStream("local.properties"))
+
+        publishing {
+            // Authenticate at the repository
+            repositories {
+                maven {
+                    url = uri("https://maven.pkg.github.com/cyface-de/android-backend")
+                    credentials {
+                        username = properties.getProperty("github.user")
+                        password = properties.getProperty("github.token")
+                    }
+                }
```

Comment:
That's the repo we publish to. A write:packages personal access token is required to do so. See Readme which is added in a later commit.